src/lib/expansion/accordion-item.ts

```diff
@@ -44,6 +52,8 @@ export class AccordionItem implements OnDestroy {
       } else {
         this.closed.emit();
       }
+
+      this._changeDetectorRef.markForCheck();
```

Comment:
Add a comment that mentions what sets `expanded` outside of the normal `@Input` binding?